src/content/zh-tw/updates/2017/10/remove-shadow-piercing.md

```diff
@@ -9,7 +9,7 @@ description: 向shadow-piercing CSS選擇器說再見。
 {# wf_featured_snippet: 向shadow-piercing CSS選擇器說再見。 #}
```

Comment:
Why non-Latin characters.

Reply:
It won't show up, `wf_featured_snippet` is only used on the EN version of files.